src/js/components/Calendar/__tests__/__snapshots__/Calendar-test.js.snap

```diff
@@ -454,7 +452,6 @@ exports[`Calendar date renders 1`] = `
           <button
             aria-label="December 2017"
             className="c7"
-            disabled={true}
```

Comment:
This seems like it has been broken.

Reply:
I've fixed this

Reply:
It doesn't appear fixed. We shouldn't be losing the fact that the button is disabled.

Also, if you use `bounds` with Calendar, they dates outside of the bounds aren't being disabled when they should be.